### ISSUE TITLE
Non Mandatory Sections 

### ISSUE THREAD

**Issue by stevekamps** [technicalbirdVayuz/volleyball-unite-usa-issue-tracker#118]
For a High School page creation under "Coaching Staff Creation" in the signup process can we please add "Skip" button at the top that says skip and complete later?  This is a lot of info that might scare high school administrators away.  Do not make any of these fields mandatory at signup.  Then once completed have a notification that tells them to complete their profile

**Comment by technicalbirdVayuz** [comment]
@rgnarok @stevekamps Can we remove it completely from the signup? It can be added later on also. Please share yours opinion

**Comment by stevekamps** [comment]
Yes please remove this from the high school organization signup pricess.  Then add the notification once they are activated.

"Complete your organization profile by adding all of your coaches information"
